api/v1alpha1/hostedcluster_types.go

```diff
@@ -21,6 +21,11 @@ type HostedClusterSpec struct {
 	// workers. It should have an ".dockerconfigjson" key containing the pull secret JSON.
 	PullSecret corev1.LocalObjectReference `json:"pullSecret"`
 
+	// AuditWebhook contains metadata for configuring an audit webhook
+	// endpoint for a cluster to process cluster audit events
+	// +optional
```

Comment:
How about some docs about on which platform types this is supported (IBM cloud?)

Reply:
done